Cargo.toml

```diff
@@ -1,21 +1,17 @@
 [package]
 name = "poseidon252"
-version = "0.1.0"
+version = "0.2.0"
 authors = [
-  "zer0 <matteo@dusk.network>"
+  "zer0 <matteo@dusk.network>", "vlopes11 <victor@dusk.network>", "CPerezz <carlos@dusk.network>"
 ]
 edition = "2018"
 
 [dependencies]
-curve25519-dalek = { version = "2", features = ["serde", "avx2_backend"] }
+dusk-bls12_381 = "0.1"
 merlin = "2"
 lazy_static = "1.3.0"
-hades252 = { git = "https://github.com/dusk-network/hades252" }
-
-[dependencies.bulletproofs]
-git = "https://github.com/dusk-network/bulletproofs"
-branch = "develop"
-features = ["yoloproofs"]
+hades252 = { git = "https://github.com/dusk-network/hades252", branch = "master" }
```

Comment:
As discussed, point to `0.5.0` already. Even if it fails the build on travis it will pass once we add the tag.